anatomy/track-tooling/analyzers/interface.md

```diff
@@ -22,31 +22,63 @@ The `analysis.json` file should be structured as followed:
 ```json
 {
   "status": "...",
+  "summary": "This solution looks good but has a few points to address",
   "comments": [
     {
       "comment": "ruby.general.some_paramaterised_message",
-      "params": { "foo": "param1", "bar": "param2" }
+      "params": { "foo": "param1", "bar": "param2" },
+      "type": "essential"
     },
     {
-      "comment": "ruby.general.some_paramaterised_message",
-      "params": {}
+      "comment": "ruby.general.some_unparamaterised_message",
+      "params": {},
+      "type": "actionable"
     },
     {
-      "comment": "ruby.general.some_paramaterised_message"
+      "comment": "ruby.general.some_unparamaterised_message"
     },
-    "ruby.general.some_paramaterised_message"
+    "ruby.general.some_unparamaterised_message"
   ]
 }
 ```
 
-Messages are keys into `website-copy/automated-comments/`, e.g. [`ruby.general.explicit_return -> automated-comments/ruby/general/explicit_return.md`](https://github.com/exercism/website-copy/blob/47af5b309ac263629ca5c52904046f81e0cc8def/automated-comments/ruby/general/explicit_return.md).
-Those markdown files can be parameterized with strings like this `Try %{variable_name} += 1 instead`.
 
-The following statuses are valid:
+### `summary` (optional)
+
+The summary field is a text field that summarises the output. 
+It might say something like "Your solution is nearly there - there's just two small changes you can make." or "The code works great, but there's a little bit of linting that needs doing.".
+This summary is rendered on the website above the comments.
+
+
+### `comments`
+Comments are keys into `website-copy/automated-comments/`, e.g. [`ruby.general.explicit_return -> automated-comments/ruby/general/explicit_return.md`](https://github.com/exercism/website-copy/blob/47af5b309ac263629ca5c52904046f81e0cc8def/automated-comments/ruby/general/explicit_return.md).
+
+Then can be structured either as single pointer strings (e.g. the last example above) or using a JSON Object to specify the follow keys:
+
+#### `comment`
+
+The pointer-string to a file in `website-copy`
+
+#### `params` (optional)
+
+A JSON Object containing any params that should be interpolated during rendering. 
+For example, in the markdown file, you could write `Try %{variable_name} += 1 instead`, and then use `params` to substitute `%{variable_name}` for the actual variable that the student used.
```

Comment:
Maybe add an example to show what that would look like? Something like `"params": { "variable_name": "foo"}`